backends/cadence/aot/compiler.py

```diff
@@ -59,15 +60,25 @@ def convert_pt2(
     """
 
     # Export with dynamo
-    model_gm = torch.export.export_for_training(model, inputs).module()
-
-    if model_gm_has_SDPA(model_gm):  # pyre-fixme[6]
+    decomp_table = torch.export.default_decompositions()
+    ops_to_keep = [
+        torch.ops.aten.conv1d.default,
+        torch.ops.aten.conv2d.default,
+        torch.ops.aten.layer_norm.default,
+        torch.ops.aten.linear.default,
+        torch.ops.aten.matmul.default,
+    ]
+    # pyre-fixme[6]: For 1st argument expected `Dict[typing.Callable[..., typing.Any
```

Comment:
pyre are disabled in ET but still used internally. We need to sort it out but not in this PR